README.md

```diff
@@ -192,6 +192,7 @@ sh -c "$(wget https://raw.githubusercontent.com/ddvk/remarkable-hacks/master/pat
 
 or if offline
 
+chmod +x /home/rmhacks/patch.sh
```

Comment:
It might be better to just change it to `sh /home/rmhacks/patch.sh`

Reply:
@Eeems You are right, tested and updated.